src/screens/LeaderboardScreen.js

```diff
@@ -13,6 +18,27 @@ export class LeaderboardScreen extends Component<Props> {
     return <Text>Hi</Text>;
   };
 
+  _renderLeaderBoard = (item, index) => {
+    return (
+      <View style={styles.leaderboard}>
+        <View style={styles.leaderboard__deckName}>
+          <TouchableOpacity
+            onPress={() =>
+              alert(`W: ${item.wins} L: ${item.games - item.wins}`)
+            }
+          >
+            <Text>
+              {index + 1}. {item.commander}
+            </Text>
+          </TouchableOpacity>
+        </View>
+        <View style={styles.leaderboard__deckElo}>
+          <Text>{Number(item.elo).toFixed(2)}</Text>
+        </View>
```

Comment:
Would it be beneficial to make that two a `const` with a variable name so you don't forget what it's for?